### PR TITLE
Add support to use authentication token in order to access kubernetes…

### PR DESCRIPTION
**Pull request by mabuaisha**
Add support to use authentication token in order to access kubernetes api using public ip

### CODE REVIEW

**Comment by mabuaisha** [comment]
@0lvin Done fixed the issue

File: CHANGELOG.txt [hunk 1]
```diff
@@ -30,3 +30,6 @@
   - Support checking statuses for Kubernetes resources.
 2.0.0.1:
   - Fix Conditional service check
+
+2.0.0.2:
+  - Add support for creating resources on Kubernetes using service account token
```
Comment (EarthmanT) [inline CHANGELOG.txt]: Change version to 2.1.0. This is a new feature!

Comment (mabuaisha) [reply CHANGELOG.txt]: Done

**Comment by EarthmanT** [comment]
This wont be 2.0.0.2, it will be 2.1.0.

File: CHANGELOG.txt (hunk 1 above)
Comment (EarthmanT) [reply CHANGELOG.txt]: 2.1.0

File: cloudify_kubernetes/k8s/config.py [hunk 2]
```diff
@@ -130,10 +130,20 @@ def _do_prepare_api(self):
...
-
+                    # Update the api_key value in order to use on the header
+                    #  api request
+                    if key == 'api_key':
+                        api_options[key] =\
+                            {"authorization":
+                                "Bearer {0}".format(api_options[key])}
+                    setattr(configuration, key, api_options[key])
+
+            configuration.debug = True
```
Comment (0lvin) [inline cloudify_kubernetes/k8s/config.py]: Maybe have sense to connect this flag to some inputs?

**Comment by mabuaisha** [comment]
@0lvin Done

**Pull request merged**